src/StudyUtils.in.jsm

```diff
@@ -402,6 +588,9 @@ class StudyUtils {
   }
 
   // telemetry from addon, mostly from webExtension message.
+  /**
+  * @async
+  */
```

Comment:
telemetry (no undercores) does schema checking.

Reply:
looks like `telemetry` forwards the data from the addon to `_telemetry` where the schema is validated